tests/reference_exports/animation_spot_light.escn

```diff
@@ -67,6 +67,7 @@ tracks/2/keys = {
 mesh = SubResource(2)
 visible = true
 transform = Transform(1.0, 0.0, 0.0, 0.0, 1.0, 0.0, 0.0, 0.0, 1.0, 0.0, 0.0, 0.0)
+material/0 = SubResource(1)
```

Comment:
Hmm. This is breaking multiple-surface materials. Nearly all of the escn's write to material/0 on the node, which is incorrect. It should be part of the mesh resource.